packages/testing/config/jest/api/jest.setup.js

```diff
@@ -31,12 +57,27 @@ const seedScenario = async (scenario) => {
 }
 
 const teardown = async () => {
-  const prismaModelNames = (await getSchemaDefinitions()).datamodel.models.map(
-    (m) => m.dbName || m.name
-  )
+  for (const modelName of teardownOrder) {
+    try {
+      await db.$executeRawUnsafe(`DELETE from "${modelName}"`)
+    } catch (e) {
+      if (e.message.match(/FOREIGN KEY constraint failed/)) {
```

Comment:
Would it be possible to compare using `instanceof`? Its probably a little more reliable than checking the message

Reply:
I feel like it was a super generic error, something like `PrismaError` so I knew I had to check the message itself (gross, I know) but I'll double check...